src/applications/disability-benefits/all-claims/sass/disability-benefits.scss

```diff
@@ -1,13 +1,13 @@
 // Variables
 
-@import "~@department-of-veterans-affairs/formation/sass/shared-variables";
```

Comment:
Can we change all of this back, looks like your editor may have gotten a little opinionated.

Reply:
@imeecuison ^^^^^^^^^^^^^

Reply:
yikes! okay!